src/control/Control.Layers.js

```diff
@@ -50,6 +50,8 @@ export const Layers = Control.extend({
 		// @option collapsed: Boolean = true
 		// If `true`, the control will be collapsed into an icon and expanded on mouse hover, touch, or keyboard activation.
 		collapsed: true,
+		// If > 0, the control will stay open for longer. This makes it easier to scroll long layer lists.

```

Comment:
This is missing some docstrings, and won't compile into the API documentation. Please follow the format for other options.

Also: what are the units? Seconds? Minutes? Milli-hours?

Reply:
done